### PR TITLE
Allow CORs for *.ft.com origins

### PR DESCRIPTION
**Pull request by seanmtracey**

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -19,8 +20,19 @@ app.use(cookieParser());
...
 
 app.use(function(req, res, next) {
-  res.header("Access-Control-Allow-Origin", "https://ft.com");
-  res.header("Access-Control-Allow-Headers", "Origin, X-Requested-With, Content-Type, Accept");
+
+  debug('ORIGIN', req.get('origin'));
+
+  if(req.get('origin') !== undefined){
+
+    if(req.get('origin').indexOf('ft.com') > -1){
```
Comment (JakeChampion) [inline app.js:28]: `ft.com.example.com` would match

Comment (seanmtracey) [reply app.js:28]: Replace with `.endsWith?`

Comment (JakeChampion) [reply app.js:28]: Yes and change the test string to `.ft.com`
```js
if(req.get('origin').endsWith('.ft.com'))
```

Comment (seanmtracey) [reply app.js:28]: Done https://github.com/ftlabs/Connected-FT/blob/master/app.js#L28

**Pull request merged**